src/assets/scripts/client/scope/ScopeModel.js

```diff
@@ -25,7 +26,18 @@ export default class ScopeModel {
          * @type {EventBus}
          * @private
          */
-        this._eventBus = EventBus;
+        this._eventBus = null;
```

Comment:
pretty sure our existing pattern is to assign this in the `constructor`.  Some day I'd love to remove these local re-assignments of the `EventBus`.  I think I had a good reason for this once, but I've seen grown to really not like it.

Reply:
@n8rzz So this is something we need to establish because we sometimes do it differently in different classes.

I'd like to greatly increase our utilization of the (awesome) ModelSourceFactory, which requires a full and proper reset and re-initialization of objects. The way I see it, we have two cases:

1. The class instance in question will be reused: We should set null values in the constructor and the `.reset()` methods. We should set default values in `.init()`. This way we can be sure a newly constructed object is the same as one to which we've done `.reset().init(data);`.

2. The class instance in question will not be reused: We should set initial values in the constructor, and any inferred values via `.init(data)` and it's submethods (such as `.initFms()` or whatever else. A `.reset()` method will not be present.

Is this a pattern you think makes sense?

Reply:
Absolutely, and (I think) that's where my head was at when we first rolled that out.  I too would love to get more object pooling happening!

Reply:
👍 Added #1386 for that. Will make the changes here.

Reply:
done